src/harness/sas.py

```diff
@@ -82,6 +93,18 @@ def DownloadFile(self, url, ssl_cert=None, ssl_key=None):
                           self._GetDefaultSasSSLCertPath(), ssl_key
                           if ssl_key else self._GetDefaultSasSSLKeyPath()))
 
+  def UpdateSasRequestUrl(self, cipher):
+    if 'ECDSA' in cipher:
+      sas_sas_active_base_url = self._sas_sas_ec_base_url
```

Comment:
Don't you need to use `self.sas_sas_active_base_url` (prepend with `self.`)?

Reply:
done, thnx.